backend/compact-connect/docs/README.md

```diff
@@ -21,17 +21,14 @@ This system includes a mock license data API that has two functions: A synchrono
 ## How to use the API bulk-upload feature
 [Back to top](#compact-connect---technical-user-guide)
 
-The data system API supports uploading of a large CSV file for asynchronous data ingest. The feature involves using two endpoints, which are described in the [Open API Specification](#open-api-specification) above. To upload a file for asynchronous data ingest perform the following steps:
-1) Export your license data to a CSV file, formatted as follows:
+## Generating a CSV export of your license data
+
+Export your license data to a CSV file, formatted as follows:
    1) The file must be a utf-8 encoded text format
-   2) The first line must be a header with column names exactly matching the field names listed in the `POST /v0/boards/:jurisdiction/licenses` request body schema.
+   2) The first line must be a header with column names exactly matching the field names listed in the `POST /v1/compacts/{compact}/jurisdictions/{jurisdiction}/licenses` request body schema.
```

Comment:
Can we link to these column names in the README? Or, link to the example licencses.csv file?

Reply:
There is an example CSV file included directly in this README, does that work? I was hoping that discussing the API spec lower in the README would be sufficient for pointing them to more detail for column names beyond that, since that is where the fields are officially specified.

Reply:
Oh yes, sorry it's odd to read unrendered! Looks good.